pos_customer_required/static/src/js/Screens/PartnerListScreen/PartnerListScreen.esm.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+/* Copyright CoopITEasy - Simon Hick <sim@coopiteasy.be>
+   License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl) */
+
+import PartnerListScreen from "point_of_sale.PartnerListScreen";
+import Registries from "point_of_sale.Registries";
+
+const PosCustomerRequiredPartnerListScreen = (OriginalPartnerListScreen) =>
+    class extends OriginalPartnerListScreen {
+        clickPartner(partner) {
```

Comment:
a comment would help to understand why this is needed. i looked at the original function and if i understand it correctly, the goal is to prevent to call `confirm()` if the partner is unselected, right? would it work by overriding `confirm()` instead to avoid re-implementing the logic?

Reply:
Both your assumptions are correct, I've moved the change to `confirm()` (and added a comment)